docs/docusaurus.config.js

```diff
@@ -59,7 +59,8 @@ const config = {
           },
           // {to: '/blog', label: 'Blog', position: 'left'},
           {
-            href: 'https://github.com/facebook/docusaurus',
+            // href: 'https://github.com/facebook/docusaurus',
+			href: 'https://github.com/TheWidlarzGroup/react-native-notificated',
```

Comment:
Please, format the code with prettier :)

Reply:
I have removed the facebook_docusaurus link and inserted spaces before rn_notificated to prettify.

Reply:
👍🏽